main/src/main/scala/sbt/internal/TaskProgress.scala

```diff
@@ -33,9 +31,11 @@ private[sbt] final class TaskProgress(currentRef: ProjectRef) extends ExecutePro
   private[this] val isAllCompleted = new AtomicBoolean(false)
   private[this] val isStopped = new AtomicBoolean(false)
 
-  override def initial: Unit = ()
+  override def initial: Unit = {
```

Comment:
It's a little confusing that this method has no parens while the other `initial` method does.

Reply:
It probably should have parens here.